File: frontend/config/supabaseClient.js
import { createClient } from '@supabase/supabase-js'

const supabaseUrl = 'https://xwxtsyhlobiyomqlxwti.supabase.co'
const supabaseAnonKey = 'eyJhbGciOiJIUzI1NiIsInR5cCI6...sscY'

export const supabase = createClient(supabaseUrl, supabaseAnonKey)

File: backend/utils/supabase_client.py
from supabase import create_client, Client
import os

SUPABASE_URL = os.getenv("SUPABASE_URL")
SUPABASE_KEY = os.getenv("SUPABASE_SERVICE_ROLE_KEY")  # updated to match your .env

if not SUPABASE_URL or not SUPABASE_KEY:
    raise ValueError("SUPABASE_URL and SUPABASE_SERVICE_ROLE_KEY must be set in your environment.")

supabase: Client = create_client(SUPABASE_URL, SUPABASE_KEY)
